mmv1/products/dialogflowcx/Environment.yaml

```diff
@@ -25,6 +25,7 @@ references: !ruby/object:Api::Resource::ReferenceLinks
     'Official Documentation': 'https://cloud.google.com/dialogflow/cx/docs'
   api: 'https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3/projects.locations.agents.environments'
 async: !ruby/object:Api::OpAsync
+  actions: ['create', 'update']
```

Comment:
It looks like we used to wait for delete too. Is that no longer needed?

Reply:
Delete was never a long-running operation for either [Version](https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3/projects.locations.agents.flows.versions/delete#response-body) or [Environment](https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3/projects.locations.agents.environments/delete#response-body). It just happened to work before, possibly because of some other code that short-circuited waiting? IDK. But yeah, it was never needed.